src/components/modals/authentification/ModalAuthConnexionView.tsx

```diff
@@ -0,0 +1,101 @@
+/**
+ * Modal avec formulaire de connexion quand user deja inscrit
+ */
+
+import { firebaseSignInUser } from '@/api/authentification';
+import { SubmitHandler, useForm } from 'react-hook-form';
+
+type Props = {
+  setContenuModal: React.Dispatch<
+    React.SetStateAction<
+      'init' | 'connexion' | 'inscription' | 'password-forget'
+    >
+  >;
+};
+
+type LoginFormTypeConnexion = {
+  email: 'string';
+  password: 'string';
+};
+
+const ModalAuthConnexionView = ({ setContenuModal }: Props) => {
+  const { handleSubmit, register, setError, reset } =
+    useForm<LoginFormTypeConnexion>();
+
+  /**
+   * 2.
+   */
+  const handleSignInUser = async ({ email, password }: LoginFormTypeConnexion) => {
+    const { error } = await firebaseSignInUser(email, password);
+    if (error) {
+      console.log("Error handleSignInUser")
+      //toast.error(error.message);
```

Comment:
Message de Sonarlint : Remove this commented out code.sonarlint(typescript:S125)

Reply:
corrigé